dkg/disk.go

```diff
@@ -127,58 +125,26 @@ func writeDepositData(aggSigs map[core.PubKey]*bls_sig.Signature, withdrawalAddr
 }
 
 // checkWrites writes sample files to check disk writes and removes sample files after verification.
-func checkWrites(dataDir string, def cluster.Definition) error {
-	var shares []share
-	sigs := make(map[core.PubKey]*bls_sig.Signature)
-	for i := 0; i < def.NumValidators; i++ {
-		tss, sks, err := tbls.GenerateTSS(def.Threshold, len(def.Operators), rand.Reader)
-		if err != nil {
-			return err
+func checkWrites(dataDir string) error {
+	checkBody := []byte("delete me: dummy file used to check write permissions")
+	files := []string{"cluster-lock.json", "deposit-data.json", "validator_keys/keystore-0.json"}
```

Comment:
you might need to support windows as well which has a different path separator. So suggest using `filepath.Join` and `filepath.Dir() != ""` to check for separator below.

Reply:
done